backend/competitorRoutes.py

```diff
@@ -43,11 +42,11 @@ def delete(self):
 
 
 class Competitor(Resource):
-    def get(self, id):
-        return competitors.findCompetitor(id)
+    def get(self, competitorId):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/brownben/munro/pullRequest?prid=4764451)